src/vorta/__main__.py

```diff
@@ -54,11 +59,44 @@ def exception_handler(type, value, tb):
         if os.fork():
             sys.exit()
 
+    # want_development is None when flag is not called
+    # it's set to the default directory when flag is called without an argument
+    # and it's set to the flag's argument if the flag has one
+    if want_development:
+        if want_development == DEFAULT_DIR_FLAG:
+            pass
+        else:
+            config.DEV_MODE_DIR = Path(want_development)
+
+        if not os.path.exists(config.DEV_MODE_DIR):
+            os.makedirs(config.DEV_MODE_DIR)
+
+        config.SETTINGS_DIR = config.DEV_MODE_DIR / 'settings'
+        if not os.path.exists(config.SETTINGS_DIR):
+            os.makedirs(config.SETTINGS_DIR)
+
+        config.LOG_DIR = config.DEV_MODE_DIR / 'logs'
+        if not os.path.exists(config.LOG_DIR):
+            os.makedirs(config.LOG_DIR)
+
+        config.CACHE_DIR = config.DEV_MODE_DIR / 'cache'
+        if not os.path.exists(config.CACHE_DIR):
+            os.makedirs(config.CACHE_DIR)
+
+        config.TEMP_DIR = config.CACHE_DIR / 'tmp'
+        if not os.path.exists(config.TEMP_DIR):
+            os.makedirs(config.TEMP_DIR)
+
+        config.PROFILE_BOOTSTRAP_FILE = config.DEV_MODE_DIR / '.vorta-init.json'
+
+    # except Exception as exception:
+    #     print("That's an invalid path for a dir \n", exception)
```

Comment:
It seems like you either forgot to remove these lines or reimplement the exception handling.